collections/_article/godot-4-3-a-shared-effort.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Godot 4.3, a shared effort"
+excerpt: "With nearly over 3,500 commits authored by over 500 contributors, the latest Godot Engine release comes packed full of new features and improvements."
```

Comment:
```suggestion
excerpt: "With over 3,500 commits authored by over 500 contributors, the latest Godot Engine release comes packed full of new features and improvements."
```
We passed the threshold after the ~25 commits I merged since rc3.